lib/galaxy/files/sources/_pyfilesystem2.py

```diff
@@ -3,6 +3,8 @@
 import logging
 import os
 
+import fs
+
```

Comment:
```suggestion
```
This seems to be unused and fails the tests.
Was this an overeager vscode auto-import ? 😆

Reply:
Mh, I use it here: `dirname = fs.path.dirname(target_path)`, line 50.

Do you think I can use os.path? I was not sure if there is any plugin that has such a strange syntax that we need `fs`.